frontend/src/containers/Home.tsx

```diff
@@ -11,32 +11,19 @@ import MarkdownRender from "../components/MarkdownRender";
 import "./Home.css";
 
 function Home() {
-  const [filter, setFilter] = useState("");
   const { homepage, loading } = usePublicHomepage();
   const { t } = useTranslation();
   const dateFormatter = useDateTimeFormatter();
 
   const onSearch = (query: string) => {
-    setFilter(query);
+    window.location.assign("/search/" + query);
```

Comment:
You should use the history hook to do this:

const history = useHistory<LocationState>();

Look at the AddChart file for reference.

Reply:
Done.